rosa-hypershift/rosa-hosted-wrapper.py

```diff
@@ -1302,6 +1304,11 @@ def main():
         type=str,
         help='Specify a desired branch of the corresponding git',
         default='master')
+    parser.add_argument(
+        '--ticket-id',
+        type=int,
+        help='Ticket ID for cloud spend approval',
+        required=True)
```

Comment:
Why do we want to enforce this as `required=True`?

Reply:
@smalleni: Having `required=True`, will compel users to provide ticket-id,
which may not be the case always, thoughts?